### PR TITLE
Make it easier for operators to debug startup failures (SOFTWARE-4981)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

**Comment by brianhlin** [comment]
Corresponding SLATE update merged here: https://github.com/slateci/slate-catalog/pull/554

File: base/etc/osg/image-config.d/20-osg-ce-setup.sh [hunk 1]
```diff
@@ -94,4 +96,6 @@ pvc_dirs+=(/var/lib/condor-ce
            /var/lib/condor-ce/spool/ceview)
 chown condor:condor ${pvc_dirs[*]}
 
-set +xe
+[[ ${HOSTED_CE_DEBUG} == 'true' ]] || set +e
```
Comment (matyasselmeci) [inline base/etc/osg/image-config.d/20-osg-ce-setup.sh:99]: This is redundant

**Comment by matyasselmeci** [comment]
so this doesn't actually display more debug info, it just makes the script keep going after failures?

File: base/etc/osg/image-config.d/20-osg-ce-setup.sh (hunk 1 above)
Comment (brianhlin) [reply base/etc/osg/image-config.d/20-osg-ce-setup.sh:99]: Oops, I meant `set +x`

**Comment by brianhlin** [comment]
Yeah, I don't really see a reason to split up bumping verbosity (something that we may add in the future) from the ability to let the CE come up regardless of failures.

**Comment by matyasselmeci** [comment]
It's just the name that's misleading -- 'debug mode' usually means more verbose.

**Comment by brianhlin** [comment]
I'm open to suggestions!

**Comment by matyasselmeci** [comment]
- HOSTED_CE_CONTINUE_ON_ERROR
- HOSTED_CE_KEEP_GOING
- HOSTED_CE_NOFAIL

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**